### ISSUE TITLE
Control buttons break on smaller screens

### ISSUE THREAD

**Issue by AhnafAhamed** [Suvink/pusher-tic-tac-toe#7]
Bug

![image](https://user-images.githubusercontent.com/42427928/198055774-24c2f0b6-a233-4b49-b208-5f21e495eda0.png)

Suggestion

![image](https://user-images.githubusercontent.com/42427928/198056112-4cbed409-9b38-4b29-bb96-f90c3caa8043.png)

I can work on this 😊

**Comment by Suvink** [comment]
Hi @AhnafAhamed, That was intentional. Do you have a better idea to place the control button than this?
What do you think of a FAB?

**Comment by AhnafAhamed** [comment]
@Suvink  yes sounds good to me, we can do something like this https://codepen.io/bkazi/pen/pgRREv with a settings icon

**Comment by Suvink** [comment]
Yes! That would be so awesome. Please go ahead @AhnafAhamed
